lookout/style/typos/metrics.py

```diff
@@ -0,0 +1,96 @@
+import copy
+from enum import Enum, unique
+import os
+import pprint
+from typing import Dict, List, Set, Tuple
+
+import jinja2
+import pandas
+from sklearn.metrics import accuracy_score, precision_recall_fscore_support
+
+from lookout.style.typos.utils import Columns
+
+
+@unique
+class ScoreMode(Enum):
+    """Modes for calculation scores of typos correction."""
+
+    detection = "detection"
+    correction = "correction"
+    on_corrected = "on_corrected"
+
+
+def _load_jinja2_template(report_template_filename: str) -> jinja2.Template:
+    env = jinja2.Environment(trim_blocks=True, lstrip_blocks=True, keep_trailing_newline=True,
+                             extensions=["jinja2.ext.do"])
```

Comment:
Sorry for misleading: since it is going to be a common function, that env update is actually required